src/pages/home/components/dropdown.module.scss

```diff
@@ -0,0 +1,54 @@
+@import "../../../assets/stylesheets/colors.scss";
+
+@keyframes growUp {
+  0% {
+    transform: scaleY(1);
+  }
+  80% {
+    transform: scaleY(1.1);
+  }
+  100% {
+    transform: scaleY(0);
+  }
+}
+
+.hideMenu {
+  display: none;
+}
+
+.showMenu {
```

Comment:
Consider reusing code, for that, separate this class into multiple clases.

like:
`.show-menu-border: { ... the attributes for the broder ... }`

 `.show-menu-layout: { ... thinks like position, sizes, margin ... }`
 
 and something similar for animation.

Reply:
Also, there is not a general rule, but consider using slugs for css classes instead of camelCase.
There are more standards for that, but there's not worth to go further in details here,